content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-30-0.md

```diff
@@ -0,0 +1,1657 @@
+
+**Introduced in**: Armory CD 2.28.6
+
+> Breaking changes are kept in this list for 3 minor versions from when the change is introduced. For example, a breaking change introduced in 2.21.0 appears in the list up to and including the 2.24.x releases. It would not appear on 2.25.x release notes.
+
+
+{{< include "breaking-changes/bc-kubectl-120.md" >}}
+{{< include "breaking-changes/bc-k8s-v2-provider-aws-iam-auth.md" >}}
+
+
+## Known issues
+<!-- Copy/paste known issues from the previous version if they're not fixed. Add new ones from OSS and Armory. If there aren't any issues, state that so readers don't think we forgot to fill out this section. -->
+
+### 1.30+ “required artifacts to bind” breaks pipelines
+Expected artifacts can be used in automated triggers and stages, and OSS [1.30](https://spinnaker.io/changelogs/1.30.0-changelog/#changes-to-the-way-artifact-constraints-on-triggers-work) changed the way artifact constraints work on triggers. Unfortunately those changes broke the previous behavior when triggering a pipeline from a stage, and this fix restores the previous behavior.
+
+**Affected versions**: Armory CD 2.30.0
+
+### Clouddriver and Spring Cloud
+The Spring Boot version has been upgraded, introducing a backwards incompatible change to the way configuration is loaded in Spinnaker. Users will need to set the ***spring.cloud.config.enabled*** property to ***true*** in the service settings of Clouddriver to preserve existing behavior. All of the other configuration blocks remain the same.
+
+**Affected versions**: Armory CD 2.30.0
+
+### Application attributes section displays “This Application has not been configured”
+There is a known issue that relates to the **Application Attributes** section under the **Config** menu. An application that was already created and configured in Spinnaker displays the message, “This application has not been configured.” While the information is missing, there is no functional impact.
+
+**Affected versions**: Armory CD 2.28.0 and later
+
+### SpEL expressions and artifact binding
+There is an issue where it appears that SpEL expressions are not being evaluated properly in artifact declarations (such as container images) for events such as the Deploy Manifest stage. What is actually happening is that an artifact binding is overriding the image value.
+
+**Workaround**:
+2.27.x or later: Disable artifact binding by adding the following parameter to the stage JSON: `enableArtifactBinding: false`.
+This setting only binds the version when the tag is missing, such as `image: nginx` without a version number.
+
+**Affected versions**: Armory CD 2.27.x and later
+
+###  Dinghy fails to start with SQL enabled
+This is a known bug with the java version
+* *enabledTLSProtocols=TLSv1.2* needs to be added as an argument on newer JVMs.
```

Comment:
```suggestion
```

This known issue relates to Early Access functionality that won't affect most users. Suggest removing to avoid concerns about stability of the service.